driver-core/src/main/java/com/datastax/driver/core/utils/MoreObjects.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.core.utils;
+
+import java.util.Arrays;
+
+/**
+ * Driver-specific implementation of utility object methods.
+ * <p>
+ * They are available in some versions of Java/Guava, but not across all versions ranges supported by the driver, hence
+ * the custom implementation.
+ */
+public class MoreObjects {
```

Comment:
Since it's public, maybe we could add javadocs for methods in this class, even if I concede that their purposes are pretty obvious.